frontend/packages/kubevirt-plugin/src/utils/validations/vm/disk.ts

```diff
@@ -139,12 +156,19 @@ export const validateDisk = (
     // TODO: implement CDROM disk bus validation
     if (disk.getType() === DiskType.DISK) {
       addRequired(disk.getDiskBus());
-      validations.diskInterface = validateBus(disk.getDiskBus(), allowedBusses);
+      validations.diskInterface = validateBusValue(disk, templateValidations);
     }
   }
   return {
     validations,
     hasAllRequiredFilled: !!hasAllRequiredFilled,
-    isValid: !!hasAllRequiredFilled && !Object.keys(validations).find((key) => validations[key]),
+    isValid:
+      !!hasAllRequiredFilled &&
+      !Object.keys(validations).find(
+        (key) =>
+          validations[key] &&
```

Comment:
the find will always return `undefined` here.

shouldn't the condition rather be? 
```tsx
validations[key] && (validations[key].validationErrorType === ValidationErrorType.Error ||  validations[key].validationErrorType === ValidationErrorType.TrivialError)
```

Reply:
this causes the tab validation to pass when it should not and allows you to go to the advanced step

Reply:
sorry typo, please see comment bellow